src/compiler/factory.ts

```diff
@@ -3182,8 +3258,7 @@ namespace ts {
                     finally { if (e) throw e.error; }
                 }
                 return ar;
-            };
-        `
+            };`

```

Comment:
Curious why all these ending backticks got moved up a line? Simple reason. I updated `guessIndentation` to infer whitespace from whitespace-only lines (but not empty lines), as when converting comments into synthetic ones (as on accessors), once you ignore the first line you can often be left with just whitespace! In any case, this brought to my attention the fact that some of our helpers had their trailing backtick on a new line tabbed one set more left than the rest of the lines in the helper. We were inconsistent about this, so I just updated them all to have the backtick after the last nonwhitespace character.